### PR TITLE
Remove delete-fork-link feature

### LINKED ISSUE #1786: Remove the `delete-fork-link` feature (Implemented by GitHub natively)
https://github.blog/changelog/2019-02-14-prompt-to-clean-up-merged-forks 🎉

Here's how it looks in a PR:

screen shot 2019-02-15 at 14 14 17

---

See a previous removal on how to do it: https://github.com/sindresorhus/refined-github/commit/e8105c631a11e7fa4022a80e2c47fa5d2145f6f8

### PR DESCRIPTION
**Pull request by dying**
Removed and disabled the `delete-fork-link` feature and file.

[This feature has been implemented by GitHub.](https://github.blog/changelog/2019-02-14-prompt-to-clean-up-merged-forks/)

Fixes #1786

### CODE REVIEW

File: source/content.css [hunk 1]
```diff
@@ -165,19 +165,6 @@ a .commit-ref:hover span {
...
-}
-
-#refined-github-delete-fork-link {
-	padding: 10px 10px 0;
-	position: absolute;
-	top: 45px;
-	right: 30px;
-	color: #df3e3e;
-}
-
```
Comment (bfred-it) [inline source/content.css]: 🙌

**Comment by bfred-it** [comment]
Excellent, thank you! 🎆

**Comment by katrinleinweber** [comment]
Their solution requires 3 clicks, though, yours only 1. Would it worth reverting https://github.com/sindresorhus/refined-github/commit/71dd6be7ba14cf237ab7fa8bd830f153fc9ce274, until they have iterated on the official version?

**Comment by bfred-it** [comment]
Or solution required a page load, a scroll, another click on "delete this repository". Not much better. If anything we can turn that dropdown into a two-click operation instead of 3 (which makes no sense). PR welcome to do that

**Comment by katrinleinweber** [comment]
I was counting from already having the page loaded to leave a `Thanks for merging` note ;-)

**Comment by bfred-it** [comment]
Thanks for contributing!

By the way, we can probably make both buttons visible to make it one-click. PR welcome to unhide the Delete Fork button via CSS only and make it look like the old text link

**Comment by katrinleinweber** [comment]
Sorry, but I have no free weekends in the foreseeable future to tinker with this nor to dig into the prerequisite skills :-(

**Pull request merged**